backend/corpora/common/entities/dataset.py

```diff
@@ -169,12 +151,10 @@ def asset_deletion(self):
             asset.delete_from_s3()
             asset.delete()
 
-    def deployment_directories_deletion(self):
-        for deployment_directory in self.deployment_directories:
-            object_names = get_cxg_bucket_path(deployment_directory)
-            logger.info(f"Deleting all files in bucket {cxg_bucket.name} under {object_names}.")
-            cxg_bucket.objects.filter(Prefix=object_names).delete()
-            self.session.delete(deployment_directory)
+    def delete_explorer_cxg_object_from_s3(self):
+        object_name = self.explorer_s3_uri.split("/")[3].split(".")[0]
```

Comment:
Seems like the `get_cxg_bucket_path` function below is a bit more robust than the above written parsing of the string. Why not use it?

Reply:
updated